www/js/components/point-card.js

```diff
@@ -18,19 +92,70 @@ export default class PointCard extends Component {
       height: '176px',
       background: 'url(' + this.props.point.image + ') center / cover'
     }
+    let seeButton = (
+      <Button colored onClick={this.onSeeMore.bind(this)}>See More</Button>
+    );
+    if (this.state.fullScreen) {
+      seeButton = (
+        <Button colored onClick={this.onSeeLess.bind(this)}>See Less</Button>
+      );
+    }
+
+    let point = this.props.point;
+    let day = this.getDays(point.hours).filter(
+      (dayEle)=>{
+        return dayMap.indexOf(dayEle.day) == (new Date()).getDay();
+      })[0]; // get the day that matches today.
+
+    // when is the service open
+    let timeDetails = (<span>{(day) ?
+        <span className="open-until"> Open until: {day.closes}</span>
+      :
+        <span className="open-until"> Not Open Today </span>
```

Comment:
This needs to be a features request if work is going to be done on it. Time in general needs a overhaul.